homeassistant/components/conversation.py

```diff
@@ -111,6 +111,12 @@ def async_register(hass, intent_type, utterances):
         'Toggle [the] [a] [an] {name}[s]',
         '[the] [a] [an] {name}[s] toggle',
     ])
+    async_register(hass, intent.INTENT_OPEN_COVER, [
```

Comment:
These should be registered by the cover component.

Reply:
The handler intent is registered in cover, this is just registering the utterances for the conversation piece

Reply:
At least we should not load them if the cover component is not loaded. We can do that by checking `hass.config.components` during startup and after that listen for `EVENT_COMPONENT_LOADED` events.